mido/midifiles/units.py

```diff
@@ -1,46 +1,73 @@
-def tick2second(tick, ticks_per_beat, tempo):
+DEFAULT_TEMPO = 500000  # microseconds per quarter note (i.e. 120 bpm in 4/4)
+DEFAULT_TICKS_PER_BEAT = 480  # ticks per quarter note
+DEFAULT_TIME_SIGNATURE = (4, 4)
+
+
+def tick2second(tick, ticks_per_beat=DEFAULT_TICKS_PER_BEAT,
+                tempo=DEFAULT_TEMPO):
     """Convert absolute time in ticks to seconds.
 
-    Returns absolute time in seconds for a chosen MIDI file time
-    resolution (ticks per beat, also called PPQN or pulses per quarter
-    note) and tempo (microseconds per beat).
+    Returns absolute time in seconds for a chosen MIDI file time resolution
+    (ticks/pulses per quarter note, also called PPQN) and tempo (microseconds
+    per quarter note).
     """
+    # Note: both tempo (microseconds) and ticks are per quarter note
+    #       thus the time signature is irrelevant
     scale = tempo * 1e-6 / ticks_per_beat
     return tick * scale
 
 
-def second2tick(second, ticks_per_beat, tempo):
+def second2tick(second, ticks_per_beat=DEFAULT_TICKS_PER_BEAT,
```

Comment:
"Convert absolute time in seconds to ticks." > "Round absolute time in seconds to ticks." IMO, or something of that sort to clarify that the conversion is lossy.